### ISSUE TITLE
KAT unit init takes a loong time

### ISSUE THREAD

**Issue by MildlyInterested** [KAT-Advanced-Medical/KAM#284]
**Arma 3 Version:** Latest profiling branch
**CBA Version:** Latest steam version
**ACE3 Version:** Lastest steam version

**Mods:**
```
- CBA_A3
- ace
- KAT
- Arma Script Profiler
- Intercept Minimal Dev
- other stuff
```

**Description:**
- the event handler that inits KAT on units takes a loong time
- As the picture below shows the KAT inits take about 8 ms to run
![image](https://user-images.githubusercontent.com/57712666/232337139-d73edcf2-b10c-4d93-a140-40adb4dc02e1.png)
- ACE medical only takes 6.5 microseconds to run, that's 1200 times faster.
![image](https://user-images.githubusercontent.com/57712666/232337180-85fe7897-f22b-4929-8ede-fe678fdf4d5d.png)
 - Looking at ACE's EH they call FUNC on init https://github.com/acemod/ACE3/blob/b78f8684b73c55191fe83026dc8b3a8c6bf5b67d/addons/medical_status/CfgEventHandlers.hpp#L19-L26
 - KAT/KAM calls COMPILE https://github.com/Tomcat-SG/KAM/blob/f6abbfa993382cb8e37b36e8399c1b6eb4e6f4e6/addons/airway/CfgEventHandlers.hpp#L8-L14 
 - I *think* that means the functions get recompiled on every unit, using up precious frames

**Steps to reproduce:**
1. Run server with Arma Script Profilers Tracy connected
2. Spawn in units with Zeus
3. Check frametimes

**Where did the issue occur?**
- Dedicated

**Additional Comments**
- Add a setting to disable KAT on all AI to avoid calling the init altogether

**Comment by MildlyInterested** [comment]
![image](https://user-images.githubusercontent.com/57712666/233406547-37263713-a22a-4508-814b-c78183c756ad.png)
 

Used to take about 8000 microseconds, now only takes about 60. Well done!

**Comment by MiszczuZPolski** [comment]
Resolved with #285
